portal-frontend/src/app/features/home/inbox/inbox-table/inbox-table.component.ts

```diff
@@ -57,4 +57,16 @@ export class InboxTableComponent {
       await this.router.navigateByUrl(link);
     }
   }
+
+  private getTableType(): string {
+    console.log(this.type);
+    switch (this.type) {
+      case 'Notice of Intent':
+        return 'NOI';
+      case 'Notification':
+        return 'NOTI';
```

Comment:
We usually use SRW for notifications. It would be great if we have the same value for integrity across application

Reply:
Maybe it's not the best time to change it. Cause the URL's are dependent on this type, and will start breaking in many other places.